app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/profile/ProfileFragment.kt

```diff
@@ -120,28 +126,41 @@ class ProfileFragment : BaseFragment<LayoutComposeBinding>() {
         onSettingsClicked: () -> Unit,
         onStatisticClicked: () -> Unit
     ) {
-        Column(
-            modifier = Modifier.padding(16.dp),
-            verticalArrangement = Arrangement.spacedBy(8.dp)
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
         ) {
-            TextWithHintCard(
-                hint = profileMapper.mapUserRole(uiStateSuccess.role),
-                label = uiStateSuccess.userName
-            )
-            NavigationIconCard(
-                iconId = R.drawable.ic_cafe,
-                labelStringId = R.string.action_profile_cafes,
-                onClick = onCafesClicked
-            )
-            NavigationIconCard(
-                iconId = R.drawable.ic_settings,
-                labelStringId = R.string.action_profile_settings,
-                onClick = onSettingsClicked
-            )
-            NavigationIconCard(
-                iconId = R.drawable.ic_statistic,
-                labelStringId = R.string.action_profile_statistic,
-                onClick = onStatisticClicked
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(16.dp),
+                verticalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                TextWithHintCard(
+                    hint = profileMapper.mapUserRole(uiStateSuccess.role),
+                    label = uiStateSuccess.userName
+                )
+                NavigationIconCard(
+                    iconId = R.drawable.ic_cafe,
+                    labelStringId = R.string.action_profile_cafes,
+                    onClick = onCafesClicked
+                )
+                NavigationIconCard(
+                    iconId = R.drawable.ic_settings,
+                    labelStringId = R.string.action_profile_settings,
+                    onClick = onSettingsClicked
+                )
+                NavigationIconCard(
+                    iconId = R.drawable.ic_statistic,
+                    labelStringId = R.string.action_profile_statistic,
+                    onClick = onStatisticClicked
+                )
+            }
+            Text(
```

Comment:
It's better to use:
```
Column {
  ...
  Space(modifier = Modifier.weight(1f))
  Text(...)
}
```

without `Box()`

Reply:
Fixed